src/NzbDrone.Core/Indexers/Rarbg/RarbgParser.cs

```diff
@@ -44,6 +49,11 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
 
             if (jsonResponse.Resource.torrent_results == null)
             {
+                if (jsonResponse.Resource.rate_limit == 1)
```

Comment:
suggest skip handling jsonResponse.Resource.error_code == 5 above and handle here instead
5 indicates a rate limit

Reply:
I’m basing the changes on the GHI, if we need to make other changes later we can.